### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by dlurton**
Adds instructions for using PartiQL in Maven and Gradle projects. #118 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,15 +10,46 @@ This repository contains an embeddable reference interpreter, test framework, an
...
-PartiQL should be considered to be in "preview" status.  It has been in use within a number of Amazon
-internal systems and an AWS product for over one year.  The behavior of the language itself is mostly stable 
-however the public API of the interpreter is slated to undergo significant improvements in the near term.
-(See the GitHub issues list for details.)
+PartiQL should be considered to be in "preview" status. It has been in use within a number of Amazon internal 
+systems and an AWS product for over one year. The behavior of the language itself is mostly stable however 
+the public API of the interpreter is slated to undergo significant improvements in the near term. (See the 
+GitHub issues list for details.)
+
+## Using In Your Project
```
Comment (tgregg) [inline README.md]: Maybe Using -> Consuming?

File: README.md [hunk 2]
```diff
@@ -10,15 +10,46 @@ This repository contains an embeddable reference interpreter, test framework, an
...
+| `org.partiql` | `partiql-lang-kotlin` | `0.1.0`
+
+
+For Maven builds, add this to your `pom.xml`:
+
+```
+<dependency>
+  <groupId>org.partiql</groupId>
+  <artifactId>partiql-lang-kotlin</artifactId>
+  <version>{version}</version>
```
Comment (tgregg) [inline README.md]: Another option is to just make this (and the gradle dependency below) explicitly `0.1.0`. We're going to have to change this file for every release anyway due to the "Recommended Version" column above.

**Comment by raganhan** [comment]
Would be a good opportunity to add a maven badge as well

```
[![Maven Central](https://maven-badges.herokuapp.com/maven-central/org.partiql/partiql-lang-kotlin/badge.svg)](https://maven-badges.herokuapp.com/maven-central/org.partiql/partiql-lang-kotlin)
```

File: README.md (hunk 2 above)
Comment (dlurton) [reply README.md]: I was trying to avoid having to update three different version numbers.

**Pull request merged**